prow/config/config_test.go

```diff
@@ -3779,15 +3785,151 @@ func TestRerunAuthConfigsGetRerunAuthConfig(t *testing.T) {
 				"istio/istio": prowapi.RerunAuthConfig{GitHubUsers: []string{"airbender"}},
 			},
 			refs:     &prowapi.Refs{Org: "istio", Repo: "istio"},
-			expected: prowapi.RerunAuthConfig{GitHubUsers: []string{"airbender"}},
+			cluster:  "",
+			expected: &prowapi.RerunAuthConfig{GitHubUsers: []string{"airbender"}},
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			d := Deck{}
+			d.RerunAuthConfigs = tc.configs
+			d.FinalizeDefaultRerunAuthConfigs()
+
+			if actual := d.GetRerunAuthConfig(tc.refs, tc.cluster); actual != nil && tc.expected != nil {
+				if !reflect.DeepEqual(*actual, *tc.expected) {
+					t.Errorf("Expected %v, got %v", tc.expected, actual)
+				}
+			} else {
+				if !reflect.DeepEqual(actual, tc.expected) {
+					t.Errorf("Expected %v, got %v", tc.expected, actual)
+				}
+			}
+		})
+	}
+}
+
+func TestDefaultRerunAuthConfigsGetRerunAuthConfig(t *testing.T) {
+	var testCases = []struct {
+		name     string
+		configs  []*DefaultRerunAuthConfigEntry
+		refs     *prowapi.Refs
+		cluster  string
+		expected *prowapi.RerunAuthConfig
+	}{
```

Comment:
Understand that these tests were adapted from the test with the old RerunAuthConfig, since new functionalities are added here, I would expected more scenarios to be tested:
- cluster field not empty
- cluster matching with wildcard
- cluster and orgRepo both not empty
- cluster not empty, orgRepo empty

Reply:
Added!